supervisor/store/git.py

```diff
@@ -102,14 +103,28 @@ async def reset(self) -> None:
             TemporaryDirectory, dir=self.sys_config.path_tmp
         )
         temp_path = Path(temp_dir.name)
-        await self._clone(temp_path)
+        try:
+            await self._clone(temp_path)
```

Comment:
`_clone()` currently acquires `self.lock`, but it's not necessary in this case. Conversely, when we do the move we should probably held a lock to avoid (theoretical) race condition.